common/lib/xmodule/xmodule/discussion_module.py

```diff
@@ -123,3 +123,9 @@ def non_editable_metadata_fields(self):
         # We may choose to enable sort_keys in the future, but while Kevin is investigating....
         non_editable_fields.extend([DiscussionDescriptor.discussion_id, DiscussionDescriptor.sort_key])
         return non_editable_fields
+
+    def student_view_data(self):
+        """
+        Returns a JSON representation of the student_view of this XModule.
+        """
+        return {self.metadata_translations['id']: self.discussion_id}
```

Comment:
Can you have the dictionary key value be simply "discussion_id"?  The value of this key is a contract between the server and the app and shouldn't be tied to other variables.